CoreOnly/Tests/FirebasePodTest/FirebasePodTest/AppDelegate.swift

```diff
@@ -36,20 +39,31 @@ class StorageExists: Storage {}
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+  func application(
+    _ application: UIApplication,
+    didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
+  ) -> Bool {
     // Override point for customization after application launch.
     return true
   }
 
   // MARK: UISceneSession Lifecycle
 
-  func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
+  func application(
+    _ application: UIApplication,
+    configurationForConnecting connectingSceneSession: UISceneSession,
+    options: UIScene.ConnectionOptions
```

Comment:
Not blocking, but can we see afterwards if we could have it wrap like this:

```
func application(_ application: UIApplication,
                 configurationForConnecting connectingSceneSession: UISceneSession,
                 option: UIScene.ConnectionOptions
) -> UISceneConfiguration
```

Seems a bit more readable having the first argument alongside the function name, I realize that wouldn't work with all functions though.

Reply:
Good catch!  Many changes look better after adding the option `--wrapparameters afterfirst`